platform/o.apache.commons.commons_io/module-auto-deps.xml

```diff
@@ -31,7 +31,7 @@
             </trigger-dependency>
             <implies>
                 <result>
-                    <module-dependency codenamebase="org.apache.commons.commons_io" spec="2.15.0"/>
```

Comment:
I think this is not necessary. My understanding is, that this file is used by the build system to replace a dependency on "org.apache.commons.io" with "org.apache.commons.commons_io". That implies, that the original value of the spec version is fine as that is the lowest required version.

Reply:
should I change it back to:
```
<module-dependency codenamebase="org.apache.commons.commons_io" spec="2.13.0"/>
```
since the PR i merged yesterday did modify the value already.

Reply:
I think that would make sense.

Reply:
added a note so that I don't update it in future by accident again